core/src/createTextMaskInputElement.js

```diff
@@ -98,8 +96,7 @@ export default function createTextMaskInputElement({
       }
 
       // `conformToMask` returns the information below: we need the `conformedValue` and we need to know whether
-      // some characters were rejected. We'll use `someCharsRejected` to know whether we should call the `onReject`
-      // callback
+      // some characters were rejected.
       const {conformedValue, meta: {someCharsRejected}} = conformToMask(safeRawValue, mask, conformToMaskConfig)
```

Comment:
Should we remove the `meta: {someCharsRejected}` from this line?

Or.. should we return that in an object from the `.update()` method... something like:

```js
{
  conformedValue: finalConformedValue,
  pipeRejection: pipeResults.rejected,
  maskRejection: someCharsRejected
}
```